### DISCUSSION TITLE
Aborted inclusion still adds node

### DISCUSSION THREAD

**Discussion by iklein99** [zwave-js/zwave-js#4782] [General]
Running version 9.6.0

Is it expected that if inclusion is aborted (receive that event from the controller), that the node is still added?  That is what I'm seeing.

I start inclusion, initiate inclusion on the device.  The device requests s2 unauthenticated.  I respond with NO security grants on purpose.  Get an aborted event, but the node is added anyway. 

Is this expected behavior
[zwavejs.log](https://github.com/zwave-js/node-zwave-js/files/9054868/zwavejs.log)
?

**Comment by AlCalzone** [comment]
Yes. Including a node into the network is handled by the stick.

Bootstrapping the node (exchanging security keys etc.) happens afterwards and is handled by Z-Wave JS. At this point the node is already part of the network.

**Comment by iklein99** [comment]
Thanks for the quick answer!
